build/check-images.js

```diff
@@ -0,0 +1,164 @@
+// XXX Consider moving this to 'flaws.js'
+// Or, something. Checking for flaws should be very different from checking
+// for images.
```

Comment:
I'm still uncertain about this. We have a file called `build/flaws.js` which is a very broad term. Checking image references makes sense to put into that file. Because it's all flaws. 

But what's coming soon is also proper image linting. That I don't think is a flaw. If the image is called "foo.png" but turns out it's actually a Zip file, I think it should throw an error because unlike flaws, we can't not let that kind of stuff ever get checked in. Or, if someone makes a PR that includes a 1GB jpeg file. 

Part of the flaw checking of image references is that it always tries to lookup the image reference to the full path of the image. E.g. `Image.findByURL('/en-US/docs/Web/image.png')` 
So if I've done that lookup anyway, I might start properly scrutinize it.

Reply:
THING IS, maybe we shouldn't check images based on them being used. 
What if someone checks in a 1GB `bomb.zip` into a PR but never inserts a `<img src="bomb.zip">` we still want to crash CI so that it throws a red flag to a potential reviewer. 
To accomplish that we might need something like `const files = Image.findAll(options);` in `cli.js` that finds ANY file with `glob` that is not `index.html`. 
To accomplish that we might need a new `.github/workflows/files.yml` which does the hard testing of corrupt images.

Reply:
Another thing I'm pushing out to another day is to check for `<img>` tags that don't have an `alt` text. 
https://github.com/mdn/yari/issues/1018
The challenge with that one is that it's going to be hard to accurately find the `<img>` tag back in the source HTML. So I might just make it a simple string flaw. 

Anyway, that means that image checking will expand.